tests/Samples.Server.Tests/BaseTest.cs

```diff
@@ -6,6 +6,7 @@
 using System;
 using GraphQL.Server.Common;
 using GraphQL.Server.Transports.AspNetCore.Common;
+using GraphQL.Server;
```

Comment:
?

Reply:
I moved the `Serializer` up a namespace level so it made sense when shared with Benchmarks project.